sdk/firstload/src/index.tsx

```diff
@@ -22,6 +21,7 @@ import type {
 } from '@highlight-run/client/src/types/types'
 import HighlightSegmentMiddleware from './integrations/segment'
 import configureElectronHighlight from './environments/electron'
+import firstloadVersion from './__generated/version'
```

Comment:
I decided to refer to the generated module directly here instead of import mapping around it, because this allows us to stop bundling the entire package.json into the bundle just to access the version in the Rollup build.